texthero/representation.py

```diff
@@ -27,101 +28,22 @@
 """
 
 
-def flatten(
-    s: Union[pd.Series, pd.Series.sparse],
-    index: pd.Index = None,
-    fill_missing_with: Any = 0.0,
-) -> pd.Series:
-    """
-    Transform a Pandas Representation Series to a "normal" (flattened) Pandas
-    Series.
-
-    The given Series should have a multiindex with first level being the
-    document and second level being individual features of that document
-    (e.g. tdidf scores per word). The flattened Series has one cell per
-    document, with the cell being a list of all the individual features of
-    that document.
-
-    Parameters
-    ----------
-    s : Sparse Pandas Series or Pandas Series
-        The multiindexed Pandas Series to flatten.
-
-    index : Pandas Index, optional, default to None
-        The index the flattened Series should have.
-
-    fill_missing_with : Any, default to 0.0
-        Value to fill the NaNs (missing values) with. This _does not_ mean
-        that existing values that are np.nan are replaced, but rather that
-        features that are not present in one document but present in others
-        are filled with fill_missing_with. See example below.
-
-
-    Examples
-    --------
-    >>> import texthero as hero
-    >>> import pandas as pd
-    >>> import numpy as np
-    >>> index = pd.MultiIndex.from_tuples(
-    ...         [("doc0", "Word1"), ("doc0", "Word3"), ("doc1", "Word2")],
-    ...          names=['document', 'word'])
-    >>> s = pd.Series([3, np.nan, 4], index=index)
-    >>> s
-    document  word 
-    doc0      Word1    3.0
-              Word3    NaN
-    doc1      Word2    4.0
-    dtype: float64
-    >>> hero.flatten(s, fill_missing_with=0.0)
-    document
-    doc0    [3.0, 0.0, nan]
-    doc1    [0.0, 4.0, 0.0]
-    dtype: object
-
+def _check_is_valid_DataFrame(df: Union[pd.DataFrame, pd.Series]) -> bool:
```

Comment:
There isn't really a reason why users would have a multiindex in their DataFrame. Thus, this function can be replaced by just `isinstance(df, pd.DataFrame)`